README.md

```diff
@@ -47,7 +47,7 @@ on [HPC](#nebari-hpc) or [Kubernetes](#nebari) with minimal DevOps overhead.
 
 **This repository details the [Nebari](https://nebari.dev/ "Official Nebari docs") (Kubernetes) version.**
 
-Not sure what to choose? Check out our [Setup Initialization](docs/source/installation/setup.md) page.
```

Comment:
Old:

https://github.com/nebari-dev/nebari/blob/8bd56f90b84c7686f45459ae8f7cf690721bd60a/docs/source/installation/setup.md?plain=1#L1-L7

The closest equivalent would be https://www.nebari.dev/docs/get-started/deploy

![Screenshot from 2023-05-10 23-04-41](https://github.com/nebari-dev/nebari/assets/6849766/efa4bbc5-1217-48be-885b-7648b49a4d5c)

LMK if you prefer this over linking to the general "Get started" page.

Reply:
Yes, I think we can link to "Choosing a deployment platform" here.

We also have a link to "Get Started" later in the page :)

Reply:
Done in https://github.com/nebari-dev/nebari/pull/1801#discussion_r1206493393.